### DISCUSSION TITLE
photo frame / screensaver option

### DISCUSSION THREAD

**Discussion by Symbiot78** [immich-app/immich#8065] [General]
hi

would love a way to use immich to populate a photoframe.
something like being able to create an album, fill in some photos and having a url that will show the images.

it could also be something like what's possible with unsplash like:

`https://source.unsplash.com/random/${1920}x${1080}?dogs`
where you essentially use a search function to show an image and refresh it at an interval.

or just a way to display all photos in an album randomly via a url.

I haven't found a way to do this in immich - but maybe I am looking in the wrong places?

**Comment by radh21301** [comment]
Look into https://github.com/3rob3/ImmichFrame
